pyartcd/pyartcd/util.py

```diff
@@ -267,3 +268,28 @@ async def is_build_permitted(version: str, data_path: str = constants.OCP_BUILD_
 
     # Fallback to default
     return True
+
+
+def get_release_controller_arch(release_stream_name):
+    arch = 'amd64'
+    streamNameComponents = release_stream_name.split('-')
+    for goArch in goArches:
+        if goArch in streamNameComponents:
+            arch = goArch
+    return arch
+
+
+def get_release_controller_url(release_stream_name):
+    arch = get_release_controller_arch(release_stream_name)
+    return f"https://{arch}.ocp.releases.ci.openshift.org"
+
+
+def log_dir_tree(path_to_dir):
```

Comment:
Just printing the folder structure could be confusing and hard to detect in a long log stream; I suggest prepending it with an explatatory message such as `Printing dir tree for <path>`

Reply:
add info logs